### PR TITLE
Update compatibility matrix for 1.11.1

### PR DESCRIPTION
**Pull request by arihantg**

### CODE REVIEW

**Comment by Dhivyaaj** [comment]
No JIRA Ids found for the PR. Jira id is mandatory to update fix version in jira.                                            Please update respective Jira id in PR title or commit message                                            if the PR is intented for default branches of repo.                                            For mandatory exemptions, comment trigger phrase 'skip jira-id-check' in PR.                                            For manual trigger, comment trigger phrase 'run jira-id-check' in PR.

**Review by DixitAakash — DISMISSED** [review]
LGTM

**Comment by akshayhavile** [comment]
skip jira-id-check

**Comment by akshayhavile** [comment]
build ako

**Review by akshayhavile — APPROVED** [review]

**Comment by Dhivyaaj** [comment]
No JIRA Ids found for the PR. Jira id is mandatory to update fix version in jira.                                            Please update respective Jira id in PR title or commit message                                            if the PR is intented for default branches of repo.                                            For mandatory exemptions, comment trigger phrase 'skip jira-id-check' in PR.                                            For manual trigger, comment trigger phrase 'run jira-id-check' in PR.

**Comment by arihantg** [comment]
build ako

**Comment by arihantg** [comment]
skip jira-id-check

File: docs/README.md [hunk 1]
```diff
@@ -89,9 +89,10 @@ AKO version 1.11.1 support for Kubernetes, Openshift, Avi Controller is as below
 
 | **Orchestrator/ Controller** | **Versions Supported** |
 | --------- | ----------- |
-| `Kubernetes` | 1.22 - 1.27 |
-| `Openshift` | 4.9 - 4.13 |
-| `Avi Controller` | 21.1.5 - 22.1.4 |
```
Comment (apalsule) [inline docs/README.md:94]: we need to support 30.1.1 here

**Comment by Dhivyaaj** [comment]
No JIRA Ids found for the PR. Jira id is mandatory to update fix version in jira.                                            Please update respective Jira id in PR title or commit message                                            if the PR is intented for default branches of repo.                                            For mandatory exemptions, comment trigger phrase 'skip jira-id-check' in PR.                                            For manual trigger, comment trigger phrase 'run jira-id-check' in PR.

**Comment by arihantg** [comment]
build ako

**Comment by arihantg** [comment]
build ako

**Comment by arihantg** [comment]
build ako

**Comment by akshayhavile** [comment]
skip jira-id-check

**Review by akshayhavile — APPROVED** [review]

**Review by DixitAakash — APPROVED** [review]

**Pull request merged**